README.md

```diff
@@ -36,3 +36,55 @@ The repository is organized this way:
 
 Presently, only the `v1` version of the formats exist.
 Hence, to update the list of providers, file a pull-request against the repository to edit `/src/links/v1/providers.json`.
+
+## Fork this repository to set up an index meta-database for your own databases
+
+- Click on the Fork button in GitHub and follow the instructions.
+
+- Edit `/src/info/v1/info.json` with your information:
+
+  - Make sure to update the URL in `available_api_versions` to point at your hosting location.
+
+- Edit `/src/info/v1/info.json` to point out your OPTiMaDe databases:
+
+  - Put all your databases on the form:
+    ```
+    {
+      "type": "child",
+      "id": "example_main",
+      "attributes": {
+        "name": "Example name",
+        "description": "Example database containing example entries",
+        "base_url": "https://www.example.com/optimade",
+        "homepage": "https://www.example.com"
+      }
+    }
+    ```
+
+  - Include also the following segment:
+    ```
+    {
+      "type": "parent",
+      "id": "optimade_index",
+      "attributes": {
+        "name": "OPTiMaDe providers",
+        "description": "OPTiMaDe index meta-database of known providers",
+        "base_url": "https://providers.optimade.org",
+        "homepage": "https://www.optimade.org"
+      }
+    }
+    ```
```

Comment:
This is new!
I like it - but it defeats our idea of "supporting" only two levels (as we write in the spec [here](https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#parent-and-child-objects)) - whatever support means in this context.
I thought the way to include the providers, was simply to include them with the type `provider`, again, as is indicated in the spec [here](https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#provider-objects).
However! I am all for not replicating too much, i.e., having the same providers spread out over all the different implementations. And having the option to direct the client to a continuously updated list of providers seems to be a good one to me.

My only issue with this then, is actually how we should present this link to the list of providers. While this may be one way, it is a bit misinformative - since the list of providers is not a parent of the index meta-database, but rather it's on the same level (in some sense).
Another option would be to introduce a fourth type, like the plural of provider, i.e., `"providers"` or a combined type of `"provider_list"`.
Another option would be to introduce a type that refers to implementation on the same "level", like `"sibling"` (to follow on with the `child` and `parent` nomenclature), but that feels a bit mis-informative as well, I feel.

So in the end, I am open to ideas, and it may even end up with what you have here. I would just like to raise the discussion to make sure it all makes sense with the rest of the spec (or adapt the spec accordingly).

Reply:
Oh... I had somehow gotten the idea that this was how it was supposed to work so that an unbroken tree was presented to the client, which could be traversed both up and down. But you are right that the present specification certainly do not mandate this setup. In particular, the fact that providers are linked out with `provider`-type objects instead of `child` already breaks this symmetry.

So, for now I should just remove this segment. But I like your version with a `provider_list` link.

Reply:
However, maybe my confusion ties into #217 a bit. Should these link objects really have these kinds of different types, or would it be better to change them to be `links`-type and put the relationship as an attribute field?